go.mod

```diff
@@ -134,14 +134,14 @@ require (
 	go.starlark.net v0.0.0-20221010140840-6bf6f0955179 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/multierr v1.9.0 // indirect
-	golang.org/x/crypto v0.6.0 // indirect
+	golang.org/x/crypto v0.14.0 // indirect
 	golang.org/x/mod v0.10.0 // indirect
-	golang.org/x/net v0.10.0 // indirect
+	golang.org/x/net v0.17.0 // indirect
```

Comment:
How indirect dependency changed if no direct dependency was changed?

Reply:
@raghavendra-talur specifically ran go get for the required version that fixes the CVE as mentioned [here](https://github.com/RamenDR/ramen/pull/1100#issuecomment-1769270195) to update the indirect dep